lib/SearchAndSort/components/DateRangeFilter/tests/DateRangeFilter-test.js

```diff
@@ -203,6 +204,22 @@ describe('DateRangeFilter', () => {
     });
   });
 
+  describe('when YYYY format is used and the provided year is invalid', () => {
+    setupApplication({
+      component: (
+        <DateRangeFilterHarness
+          dateFormat="YYYY"
+          selectedDates={{
+            startDate: '',
+            endDate: '200a',
```

Comment:
Instance dates could have non-digit characters inside them, so in our case the component should support dates like `199u` which represent years 1990-1999
Maybe we could add a new prop for validating a single date and use it inside `validateDateRange` util.

Reply:
We only need to support digits.